demo-server/src/main/java/com/illia/server/controller/DemoServerController.java

```diff
@@ -0,0 +1,54 @@
+package com.illia.server.controller;
+
+
+import com.illia.server.request_processor.RequestProcessor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.fileupload.disk.DiskFileItem;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.nio.file.Files;
+
+
+@Slf4j
+@RestController
+@RequestMapping("demo")
+public class DemoServerController {
+
+    @Autowired
+    RequestProcessor requestProcessor;
+
+    @PostMapping(value = "/uploadMultipartFile", consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
+    public ResponseEntity<String> uploadMultipartFile(@RequestParam(name = "fileName") String fileName,
```

Comment:
What for is the support of both File and MultipartFile? Remove if any of them is not used in client. 
If Multipart is used for debugging, use it and change client request to send MultipartFile as well.